util/bitarray: (unknown) failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/19942d06f60abbc20175fb456f8e718b05a75b7f

Parameters:
```
TAGS=
GOFLAGS=
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stressrace instead of stress and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
make stress TESTS=(unknown) PKG=github.com/cockroachdb/cockroach/pkg/util/bitarray TESTTIMEOUT=5m STRESSFLAGS='-stderr=false -maxtime 20m -timeout 10m'
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=894372&tab=buildLog

```
GOPATH set to /go
Running make with -j8
GOPATH set to /go
Detected change in build system. Rebooting Make.
Running make with -j8
GOPATH set to /go
cd /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc && autoconf
git submodule update --init --recursive
go install -v ./pkg/cmd/prereqs
mkdir -p pkg/sql/parser/gen
awk -f pkg/sql/parser/help.awk < pkg/sql/parser/sql.y > pkg/sql/parser/help_messages.go.tmp || rm pkg/sql/parser/help_messages.go.tmp
set -euo pipefail; \
TYPES=$(awk '/func.*sqlSymUnion/ {print $(NF - 1)}' pkg/sql/parser/sql.y | sed -e 's/[]\/$*.^|[]/\\&/g' | tr '\n' '|' | sed -E '$s/.$//'); \
sed -E "s_(type|token) <($TYPES)>_\1 <union> /* <\2> */_" < pkg/sql/parser/sql.y | \
awk -f pkg/sql/parser/replace_help_rules.awk | \
sed -Ee 's,//.*$,,g;s,/[*]([^*]|[*][^/])*[*]/, ,g;s/ +$//g' > pkg/sql/parser/gen/sql-gen.y.tmp || rm pkg/sql/parser/gen/sql-gen.y.tmp
mv -f pkg/sql/parser/help_messages.go.tmp pkg/sql/parser/help_messages.go
gofmt -s -w pkg/sql/parser/help_messages.go
awk -f pkg/sql/parser/all_keywords.awk < pkg/sql/parser/sql.y > pkg/sql/lex/keywords.go.tmp || rm pkg/sql/lex/keywords.go.tmp
awk -f pkg/sql/parser/reserved_keywords.awk < pkg/sql/parser/sql.y > pkg/sql/lex/reserved_keywords.go.tmp || rm pkg/sql/lex/reserved_keywords.go.tmp
mv -f pkg/sql/lex/keywords.go.tmp pkg/sql/lex/keywords.go
mv -f pkg/sql/lex/reserved_keywords.go.tmp pkg/sql/lex/reserved_keywords.go
gofmt -s -w pkg/sql/lex/reserved_keywords.go
mv -f pkg/sql/parser/gen/sql-gen.y.tmp pkg/sql/parser/gen/sql-gen.y
gofmt -s -w pkg/sql/lex/keywords.go
mv -f pkg/sql/parser/helpmap_test.go.tmp pkg/sql/parser/helpmap_test.go
gofmt -s -w pkg/sql/parser/helpmap_test.go
mkdir -p bin
touch bin/.submodules-initialized
go install -v uptodate
go install -v protoc-gen-gogoroach
go install -v optgen
bin/prereqs ./pkg/cmd/uptodate > bin/uptodate.d.tmp
bin/prereqs ./pkg/cmd/protoc-gen-gogoroach > bin/protoc-gen-gogoroach.d.tmp
bin/prereqs ./pkg/sql/opt/optgen/cmd/optgen > bin/optgen.d.tmp
mv -f bin/uptodate.d.tmp bin/uptodate.d
mv -f bin/optgen.d.tmp bin/optgen.d
mv -f bin/protoc-gen-gogoroach.d.tmp bin/protoc-gen-gogoroach.d
rm -rf /go/native/x86_64-pc-linux-gnu/jemalloc
mkdir -p /go/native/x86_64-pc-linux-gnu/jemalloc
cd /go/native/x86_64-pc-linux-gnu/jemalloc && /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/configure   --enable-prof
checking for xsltproc... false
checking for gcc... no
checking for cc... cc
optgen -out pkg/sql/opt/memo/expr.og.go exprs pkg/sql/opt/ops/*.opt
optgen -out pkg/sql/opt/operator.og.go ops pkg/sql/opt/ops/*.opt
optgen -out pkg/sql/opt/xform/explorer.og.go explorer pkg/sql/opt/ops/*.opt pkg/sql/opt/xform/rules/*.opt
optgen -out pkg/sql/opt/norm/factory.og.go factory pkg/sql/opt/ops/*.opt pkg/sql/opt/norm/rules/*.opt
optgen -out pkg/sql/opt/rule_name.og.go rulenames pkg/sql/opt/ops/*.opt pkg/sql/opt/norm/rules/*.opt pkg/sql/opt/xform/rules/*.opt
checking whether the C compiler works... rm -rf /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos
rm -rf /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl
mkdir -p /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos
mkdir -p /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl
build/werror.sh /go/native/x86_64-pc-linux-gnu/protobuf/protoc -Ipkg:./vendor/github.com/gogo/protobuf:./vendor/github.com/gogo/protobuf/protobuf --cpp_out=lite:/go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos ./pkg/roachpb/data.proto ./pkg/roachpb/internal.proto ./pkg/roachpb/metadata.proto ./pkg/storage/engine/enginepb/file_registry.proto ./pkg/storage/engine/enginepb/mvcc.proto ./pkg/storage/engine/enginepb/mvcc3.proto ./pkg/storage/engine/enginepb/rocksdb.proto ./pkg/util/hlc/legacy_timestamp.proto ./pkg/util/hlc/timestamp.proto ./pkg/util/unresolved_addr.proto
yes
checking for C compiler default output file name... a.out
checking for suffix of executables... sed -i -E '/gogoproto/d' /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/roachpb/data.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/roachpb/internal.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/roachpb/metadata.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/file_registry.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/mvcc.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/mvcc3.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/rocksdb.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/util/hlc/legacy_timestamp.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/util/hlc/timestamp.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/util/unresolved_addr.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/roachpb/data.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/roachpb/internal.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/roachpb/metadata.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/file_registry.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/mvcc.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/mvcc3.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/storage/engine/enginepb/rocksdb.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/util/hlc/legacy_timestamp.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/util/hlc/timestamp.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protos/util/unresolved_addr.pb.cc
build/werror.sh /go/native/x86_64-pc-linux-gnu/protobuf/protoc -Ipkg:./vendor/github.com/gogo/protobuf:./vendor/github.com/gogo/protobuf/protobuf --cpp_out=lite:/go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl ./pkg/ccl/baseccl/encryption_options.proto ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.proto ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.proto
touch bin/.cpp_protobuf_sources

sed -i -E '/gogoproto/d' /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl/ccl/baseccl/encryption_options.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl/ccl/storageccl/engineccl/enginepbccl/stats.pb.h /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl/ccl/baseccl/encryption_options.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.cc /go/src/github.com/cockroachdb/cockroach/c-deps/libroach/protosccl/ccl/storageccl/engineccl/enginepbccl/stats.pb.cc
checking whether we are cross compiling... touch bin/.cpp_ccl_protobuf_sources
no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking whether compiler is cray... no
checking whether compiler supports -std=gnu11... Scanning dependencies of target roach
yes
[  2%] Building CXX object CMakeFiles/roach.dir/batch.cc.o
[  5%] Building CXX object CMakeFiles/roach.dir/db.cc.o
checking whether compiler supports -Wall... [  7%] Building CXX object CMakeFiles/roach.dir/encoding.cc.o
[ 10%] Building CXX object CMakeFiles/roach.dir/comparator.cc.o
[ 12%] Building CXX object CMakeFiles/roach.dir/chunked_buffer.cc.o
yes
checking whether compiler supports -Werror=declaration-after-statement... [ 15%] Building CXX object CMakeFiles/roach.dir/engine.cc.o
yes
[ 17%] Building CXX object CMakeFiles/roach.dir/iterator.cc.o
checking whether compiler supports -Wshorten-64-to-32... [ 20%] Building CXX object CMakeFiles/roach.dir/file_registry.cc.o
[ 22%] Building CXX object CMakeFiles/roach.dir/mvcc.cc.o
[ 25%] Building CXX object CMakeFiles/roach.dir/merge.cc.o
[ 27%] Building CXX object CMakeFiles/roach.dir/options.cc.o
yes
checking whether compiler supports -Wsign-compare... [ 30%] Building CXX object CMakeFiles/roach.dir/snapshot.cc.o
[ 32%] Building CXX object CMakeFiles/roach.dir/timebound.cc.o
[ 35%] Building CXX object CMakeFiles/roach.dir/protos/roachpb/data.pb.cc.o
yes
[ 37%] Building CXX object CMakeFiles/roach.dir/protos/roachpb/metadata.pb.cc.o
[ 40%] Building CXX object CMakeFiles/roach.dir/protos/storage/engine/enginepb/mvcc.pb.cc.o
checking whether compiler supports -pipe... [ 42%] Building CXX object CMakeFiles/roach.dir/protos/roachpb/internal.pb.cc.o
[ 45%] Building CXX object CMakeFiles/roach.dir/protos/storage/engine/enginepb/mvcc3.pb.cc.o
yes
[ 47%] Building CXX object CMakeFiles/roach.dir/protos/storage/engine/enginepb/file_registry.pb.cc.o
checking whether compiler supports -g3... [ 50%] Building CXX object CMakeFiles/roach.dir/protos/util/hlc/legacy_timestamp.pb.cc.o
[ 55%] Building CXX object CMakeFiles/roach.dir/protos/util/hlc/timestamp.pb.cc.o
[ 55%] Building CXX object CMakeFiles/roach.dir/protos/util/unresolved_addr.pb.cc.o
[ 57%] Building CXX object CMakeFiles/roach.dir/protos/storage/engine/enginepb/rocksdb.pb.cc.o
[ 60%] Building CXX object CMakeFiles/roach.dir/rocksdbutils/env_encryption.cc.o
yes
checking how to run the C preprocessor... [ 62%] Linking CXX static library libroach.a
cc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... Scanning dependencies of target roachccl
[ 65%] Building CXX object CMakeFiles/roachccl.dir/ccl/crypto_utils.cc.o
[ 67%] Building CXX object CMakeFiles/roachccl.dir/ccl/ctr_stream.cc.o
[ 70%] Building CXX object CMakeFiles/roachccl.dir/ccl/key_manager.cc.o
[ 72%] Building CXX object CMakeFiles/roachccl.dir/ccl/db.cc.o
[ 75%] Building CXX object CMakeFiles/roachccl.dir/protosccl/ccl/baseccl/encryption_options.pb.cc.o
[ 77%] Building CXX object CMakeFiles/roachccl.dir/protosccl/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.cc.o
[ 80%] Building CXX object CMakeFiles/roachccl.dir/protosccl/ccl/storageccl/engineccl/enginepbccl/stats.pb.cc.o
[ 82%] Linking CXX static library libroachccl.a
yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking whether byte ordering is bigendian... no
checking size of void *... 8
checking size of int... 4
checking size of long... 8
checking size of long long... 8
checking size of intmax_t... 8
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking whether pause instruction is compilable... yes
checking for ar... ar
checking malloc.h usability... yes
checking malloc.h presence... yes
checking for malloc.h... yes
checking whether malloc_usable_size definition can use const argument... no
checking for library containing log... -lm
checking whether __attribute__ syntax is compilable... yes
checking whether compiler supports -fvisibility=hidden... yes
checking whether compiler supports -Werror... yes
checking whether compiler supports -herror_on_warning... no
checking whether tls_model attribute is compilable... yes
checking whether compiler supports -Werror... yes
checking whether compiler supports -herror_on_warning... no
checking whether alloc_size attribute is compilable... no
checking whether compiler supports -Werror... yes
checking whether compiler supports -herror_on_warning... touch bin/.bootstrap
find ./pkg -name node_modules -prune -o -type f -name '*.pb.go' -exec rm {} +
find ./pkg -name node_modules -prune -o -type f -name '*.pb.gw.go' -exec rm {} +
set -euo pipefail; \
  ret=$(cd pkg/sql/parser/gen && goyacc -p sql -o sql.go.tmp sql-gen.y); \
  if expr "$ret" : ".*conflicts" >/dev/null; then \
    echo "$ret"; exit 1; \
  fi
stringer -output=pkg/sql/opt/rule_name_string.go -type=RuleName pkg/sql/opt/rule_name.go pkg/sql/opt/rule_name.og.go
no
checking whether format(gnu_printf, ...) attribute is compilable... set -e; for dir in ./pkg/acceptance/cluster/ ./pkg/build/ ./pkg/ccl/backupccl/ ./pkg/ccl/baseccl/ ./pkg/ccl/storageccl/engineccl/enginepbccl/ ./pkg/ccl/utilccl/licenseccl/ ./pkg/config/ ./pkg/gossip/ ./pkg/internal/client/ ./pkg/jobs/jobspb/ ./pkg/roachpb/ ./pkg/rpc/ ./pkg/server/diagnosticspb/ ./pkg/server/serverpb/ ./pkg/server/status/ ./pkg/settings/cluster/ ./pkg/sql/distsqlrun/ ./pkg/sql/pgwire/pgerror/ ./pkg/sql/sqlbase/ ./pkg/sql/stats/ ./pkg/storage/ ./pkg/storage/closedts/ctpb/ ./pkg/storage/engine/enginepb/ ./pkg/storage/storagebase/ ./pkg/ts/tspb/ ./pkg/util/ ./pkg/util/hlc/ ./pkg/util/log/ ./pkg/util/metric/ ./pkg/util/protoutil/ ./pkg/util/tracing/; do \
  build/werror.sh /go/native/x86_64-pc-linux-gnu/protobuf/protoc -Ipkg:./vendor/github.com:./vendor/github.com/gogo/protobuf:./vendor/github.com/gogo/protobuf/protobuf:./vendor/go.etcd.io:./vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --gogoroach_out=Mgoogle/api/annotations.proto=github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/api,Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,,plugins=grpc,import_prefix=github.com/cockroachdb/cockroach/pkg/:./pkg $dir/*.proto; \
done
build/werror.sh /go/native/x86_64-pc-linux-gnu/protobuf/protoc -Ipkg:./vendor/github.com:./vendor/github.com/gogo/protobuf:./vendor/github.com/gogo/protobuf/protobuf:./vendor/go.etcd.io:./vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true,request_context=true:./pkg ./pkg/server/serverpb/admin.proto ./pkg/server/serverpb/status.proto ./pkg/server/serverpb/authentication.proto
no
checking whether compiler supports -Werror... yes
checking whether compiler supports -herror_on_warning... no
checking whether format(printf, ...) attribute is compilable... build/werror.sh /go/native/x86_64-pc-linux-gnu/protobuf/protoc -Ipkg:./vendor/github.com:./vendor/github.com/gogo/protobuf:./vendor/github.com/gogo/protobuf/protobuf:./vendor/go.etcd.io:./vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true,request_context=true:./pkg ./pkg/ts/tspb/timeseries.proto
yes
checking for a BSD-compatible install... sed -i -E 's!golang.org/x/net/context!context!g' ./pkg/server/serverpb/admin.pb.gw.go ./pkg/server/serverpb/status.pb.gw.go ./pkg/server/serverpb/authentication.pb.gw.go ./pkg/ts/tspb/timeseries.pb.gw.go
gofmt -s -w ./pkg/server/serverpb/admin.pb.gw.go ./pkg/server/serverpb/status.pb.gw.go ./pkg/server/serverpb/authentication.pb.gw.go ./pkg/ts/tspb/timeseries.pb.gw.go
/usr/bin/install -c
checking for ranlib... ranlib
checking for ld... /usr/bin/ld
checking for autoconf... /usr/bin/autoconf
checking for memalign... goimports -w ./pkg/server/serverpb/admin.pb.gw.go ./pkg/server/serverpb/status.pb.gw.go ./pkg/server/serverpb/authentication.pb.gw.go ./pkg/ts/tspb/timeseries.pb.gw.go
yes
checking for valloc... yes
checking whether compiler supports -O3... yes
checking whether compiler supports -funroll-loops... yes
checking unwind.h usability... yes
checking unwind.h presence... yes
checking for unwind.h... yes
touch bin/.gw_protobuf_sources
checking for _Unwind_Backtrace in -lgcc... yes
checking configured backtracing method... libgcc
checking for sbrk... yes
checking whether utrace(2) is compilable... no
checking whether valgrind is compilable... no
checking whether a program using __builtin_unreachable is compilable... yes
checking whether a program using __builtin_ffsl is compilable... yes
checking LG_PAGE... 12
Missing VERSION file, and unable to generate it; creating bogus VERSION
checking pthread.h usability... yes
checking pthread.h presence... yes
checking for pthread.h... yes
checking for pthread_create in -lpthread... yes
checking whether pthread_atfork(3) is compilable... yes
checking for library containing clock_gettime... none required
checking whether clock_gettime(CLOCK_MONOTONIC_COARSE, ...) is compilable... yes
checking whether clock_gettime(CLOCK_MONOTONIC, ...) is compilable... yes
checking whether mach_absolute_time() is compilable... no
checking whether compiler supports -Werror... yes
checking whether syscall(2) is compilable... yes
checking for secure_getenv... yes
checking for issetugid... no
checking for _malloc_thread_cleanup... no
checking for _pthread_mutex_init_calloc_cb... no
checking for TLS... yes
checking whether C11 atomics is compilable... yes
checking whether atomic(9) is compilable... no
checking whether Darwin OSAtomic*() is compilable... no
checking whether madvise(2) is compilable... yes
checking whether madvise(..., MADV_FREE) is compilable... no
checking whether madvise(..., MADV_DONTNEED) is compilable... yes
checking whether madvise(..., MADV_[NO]HUGEPAGE) is compilable... yes
checking whether to force 32-bit __sync_{add,sub}_and_fetch()... no
checking whether to force 64-bit __sync_{add,sub}_and_fetch()... no
checking for __builtin_clz... yes
checking whether Darwin os_unfair_lock_*() is compilable... no
checking whether Darwin OSSpin*() is compilable... no
checking whether glibc malloc hook is compilable... yes
checking whether glibc memalign hook is compilable... yes
checking whether pthreads adaptive mutexes is compilable... yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... yes
configure: creating ./config.status
config.status: creating Makefile
config.status: creating jemalloc.pc
config.status: creating doc/html.xsl
config.status: creating doc/manpages.xsl
config.status: creating doc/jemalloc.xml
config.status: creating include/jemalloc/jemalloc_macros.h
config.status: creating include/jemalloc/jemalloc_protos.h
config.status: creating include/jemalloc/jemalloc_typedefs.h
config.status: creating include/jemalloc/internal/jemalloc_internal.h
config.status: creating test/test.sh
config.status: creating test/include/test/jemalloc_test.h
config.status: creating config.stamp
config.status: creating bin/jemalloc-config
config.status: creating bin/jemalloc.sh
config.status: creating bin/jeprof
config.status: creating include/jemalloc/jemalloc_defs.h
config.status: creating include/jemalloc/internal/jemalloc_internal_defs.h
config.status: creating test/include/test/jemalloc_test_defs.h
config.status: executing include/jemalloc/internal/private_namespace.h commands
config.status: executing include/jemalloc/internal/private_unnamespace.h commands
config.status: executing include/jemalloc/internal/public_symbols.txt commands
config.status: executing include/jemalloc/internal/public_namespace.h commands
config.status: executing include/jemalloc/internal/public_unnamespace.h commands
config.status: executing include/jemalloc/internal/size_classes.h commands
(echo "// Code generated by goyacc. DO NOT EDIT."; \
 echo "// GENERATED FILE DO NOT EDIT"; \
 cat pkg/sql/parser/gen/sql.go.tmp | \
 sed -E 's/^const ([A-Z][_A-Z0-9]*) =.*$/const \1 = lex.\1/g') > pkg/sql/parser/sql.go.tmp || rm pkg/sql/parser/sql.go.tmp
(echo "// Code generated by make. DO NOT EDIT."; \
 echo "// GENERATED FILE DO NOT EDIT"; \
 echo; \
 echo "package lex"; \
 echo; \
 grep '^const [A-Z][_A-Z0-9]* ' pkg/sql/parser/gen/sql.go.tmp) > pkg/sql/lex/tokens.go.tmp || rm pkg/sql/lex/tokens.go.tmp
mv -f pkg/sql/lex/tokens.go.tmp pkg/sql/lex/tokens.go
mv -f pkg/sql/parser/sql.go.tmp pkg/sql/parser/sql.go
goimports -w pkg/sql/parser/sql.go
config.status: executing include/jemalloc/jemalloc_protos_jet.h commands
config.status: executing include/jemalloc/jemalloc_rename.h commands
config.status: executing include/jemalloc/jemalloc_mangle.h commands
config.status: executing include/jemalloc/jemalloc_mangle_jet.h commands
config.status: executing include/jemalloc/jemalloc.h commands
===============================================================================
jemalloc version   : 0.0.0-0-g0000000000000000000000000000000000000000
library revision   : 2

CONFIG             : --enable-prof CFLAGS=-g1 LDFLAGS=
CC                 : cc
CONFIGURE_CFLAGS   : -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops
SPECIFIED_CFLAGS   : -g1
EXTRA_CFLAGS       : 
CPPFLAGS           : -D_GNU_SOURCE -D_REENTRANT
LDFLAGS            : 
EXTRA_LDFLAGS      : 
LIBS               : -lm  -lgcc -lm -lpthread
RPATH_EXTRA        : 

XSLTPROC           : false
XSLROOT            : 

PREFIX             : /usr/local
BINDIR             : /usr/local/bin
DATADIR            : /usr/local/share
INCLUDEDIR         : /usr/local/include
LIBDIR             : /usr/local/lib
MANDIR             : /usr/local/share/man

srcroot            : /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/
abs_srcroot        : /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/
objroot            : 
abs_objroot        : /go/native/x86_64-pc-linux-gnu/jemalloc/

JEMALLOC_PREFIX    : 
JEMALLOC_PRIVATE_NAMESPACE
                   : je_
install_suffix     : 
malloc_conf        : 
autogen            : 0
cc-silence         : 1
debug              : 0
code-coverage      : 0
stats              : 1
prof               : 1
prof-libunwind     : 0
prof-libgcc        : 1
prof-gcc           : 0
tcache             : 1
thp                : 1
fill               : 1
utrace             : 0
valgrind           : 0
xmalloc            : 0
munmap             : 0
lazy_lock          : 0
tls                : 1
cache-oblivious    : 1
===============================================================================
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/jemalloc.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/jemalloc.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/arena.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/arena.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/atomic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/atomic.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/base.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/base.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/chunk.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/chunk.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/bitmap.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/bitmap.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/chunk_dss.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/chunk_dss.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/chunk_mmap.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/chunk_mmap.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/ckh.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/ckh.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/ctl.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/ctl.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/extent.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/extent.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/hash.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/hash.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/huge.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/huge.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/mb.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/mb.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/mutex.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/mutex.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/nstime.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/nstime.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/pages.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/pages.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/prng.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/prng.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/prof.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/prof.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/quarantine.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/quarantine.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/stats.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/stats.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/rtree.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/rtree.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/spin.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/spin.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/tcache.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/tcache.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/ticker.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/ticker.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/tsd.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/tsd.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/util.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/util.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/witness.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/witness.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/jemalloc.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/jemalloc.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/arena.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/arena.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/atomic.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/atomic.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/base.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/base.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/bitmap.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/bitmap.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/chunk.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/chunk.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/chunk_dss.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/chunk_dss.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/chunk_mmap.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/chunk_mmap.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/ckh.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/ckh.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/ctl.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/ctl.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/extent.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/extent.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/hash.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/hash.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/huge.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/huge.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/mb.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/mb.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/mutex.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/mutex.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/nstime.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/nstime.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/pages.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/pages.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/prng.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/prng.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/prof.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/prof.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/quarantine.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/quarantine.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/rtree.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/rtree.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/stats.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/stats.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/spin.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/spin.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/ticker.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/ticker.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/tcache.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/tcache.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/tsd.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/tsd.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/util.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/util.c
cc -std=gnu11 -Wall -Werror=declaration-after-statement -Wshorten-64-to-32 -Wsign-compare -pipe -g3 -fvisibility=hidden -O3 -funroll-loops -g1 -fPIC -DPIC -c -D_GNU_SOURCE -D_REENTRANT -I/go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/include -Iinclude -o src/witness.pic.o /go/src/github.com/cockroachdb/cockroach/c-deps/jemalloc/src/witness.c
ar crus lib/libjemalloc.a src/jemalloc.o src/arena.o src/atomic.o src/base.o src/bitmap.o src/chunk.o src/chunk_dss.o src/chunk_mmap.o src/ckh.o src/ctl.o src/extent.o src/hash.o src/huge.o src/mb.o src/mutex.o src/nstime.o src/pages.o src/prng.o src/prof.o src/quarantine.o src/rtree.o src/stats.o src/spin.o src/tcache.o src/ticker.o src/tsd.o src/util.o src/witness.o
ar crus lib/libjemalloc_pic.a src/jemalloc.pic.o src/arena.pic.o src/atomic.pic.o src/base.pic.o src/bitmap.pic.o src/chunk.pic.o src/chunk_dss.pic.o src/chunk_mmap.pic.o src/ckh.pic.o src/ctl.pic.o src/extent.pic.o src/hash.pic.o src/huge.pic.o src/mb.pic.o src/mutex.pic.o src/nstime.pic.o src/pages.pic.o src/prng.pic.o src/prof.pic.o src/quarantine.pic.o src/rtree.pic.o src/stats.pic.o src/spin.pic.o src/tcache.pic.o src/ticker.pic.o src/tsd.pic.o src/util.pic.o src/witness.pic.o
sed -i '/import _/d' ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
sed -i -E 's!import (fmt|math) "github.com/cockroachdb/cockroach/pkg/(fmt|math)"! !g' ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
sed -i -E 's!github\.com/cockroachdb/cockroach/pkg/(etcd)!go.etcd.io/\1!g' ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
sed -i -E 's!cockroachdb/cockroach/pkg/(prometheus/client_model)!\1/go!g' ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
sed -i -E 's!github.com/cockroachdb/cockroach/pkg/(bytes|encoding/binary|errors|fmt|io|math|github\.com|(google\.)?golang\.org)!\1!g' ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
sed -i -E 's!golang.org/x/net/context!context!g' ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
gofmt -s -w ./pkg/acceptance/cluster/testconfig.pb.go ./pkg/build/info.pb.go ./pkg/ccl/backupccl/backup.pb.go ./pkg/ccl/baseccl/encryption_options.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/key_registry.pb.go ./pkg/ccl/storageccl/engineccl/enginepbccl/stats.pb.go ./pkg/ccl/utilccl/licenseccl/license.pb.go ./pkg/config/system.pb.go ./pkg/config/zone.pb.go ./pkg/gossip/gossip.pb.go ./pkg/internal/client/lease.pb.go ./pkg/jobs/jobspb/jobs.pb.go ./pkg/roachpb/api.pb.go ./pkg/roachpb/app_stats.pb.go ./pkg/roachpb/data.pb.go ./pkg/roachpb/errors.pb.go ./pkg/roachpb/internal.pb.go ./pkg/roachpb/internal_raft.pb.go ./pkg/roachpb/io-formats.pb.go ./pkg/roachpb/metadata.pb.go ./pkg/rpc/heartbeat.pb.go ./pkg/server/diagnosticspb/diagnostics.pb.go ./pkg/server/serverpb/admin.pb.go ./pkg/server/serverpb/authentication.pb.go ./pkg/server/serverpb/init.pb.go ./pkg/server/serverpb/status.pb.go ./pkg/server/status/status.pb.go ./pkg/settings/cluster/cluster_version.pb.go ./pkg/sql/distsqlrun/api.pb.go ./pkg/sql/distsqlrun/data.pb.go ./pkg/sql/distsqlrun/processors.pb.go ./pkg/sql/distsqlrun/stats.pb.go ./pkg/sql/pgwire/pgerror/errors.pb.go ./pkg/sql/sqlbase/encoded_datum.pb.go ./pkg/sql/sqlbase/join_type.pb.go ./pkg/sql/sqlbase/privilege.pb.go ./pkg/sql/sqlbase/structured.pb.go ./pkg/sql/stats/histogram.pb.go ./pkg/storage/api.pb.go ./pkg/storage/closedts/ctpb/entry.pb.go ./pkg/storage/engine/enginepb/file_registry.pb.go ./pkg/storage/engine/enginepb/mvcc.pb.go ./pkg/storage/engine/enginepb/mvcc3.pb.go ./pkg/storage/engine/enginepb/rocksdb.pb.go ./pkg/storage/lease_status.pb.go ./pkg/storage/liveness.pb.go ./pkg/storage/log.pb.go ./pkg/storage/raft.pb.go ./pkg/storage/storagebase/proposer_kv.pb.go ./pkg/storage/storagebase/state.pb.go ./pkg/ts/tspb/timeseries.pb.go ./pkg/util/hlc/legacy_timestamp.pb.go ./pkg/util/hlc/timestamp.pb.go ./pkg/util/log/log.pb.go ./pkg/util/metric/metric.pb.go ./pkg/util/protoutil/clone.pb.go ./pkg/util/tracing/recorded_span.pb.go ./pkg/util/unresolved_addr.pb.go
touch bin/.go_protobuf_sources
go test  -exec 'stress -maxruns 100 -maxfails 1 -stderr' -tags ' make x86_64_pc_linux_gnu' -ldflags '-X github.com/cockroachdb/cockroach/pkg/build.typ=development -extldflags "" -X "github.com/cockroachdb/cockroach/pkg/build.tag=v2.1.0-beta.20180904-338-g19942d0" -X "github.com/cockroachdb/cockroach/pkg/build.rev=19942d06f60abbc20175fb456f8e718b05a75b7f" -X "github.com/cockroachdb/cockroach/pkg/build.cgoTargetTriple=x86_64-pc-linux-gnu"  ' -run "." -timeout 0 github.com/cockroachdb/cockroach/pkg/util/bitarray  -v -args -test.timeout 40m
Makefile:835: recipe for target 'stress' failed

```

--- a/pkg/cmd/teamcity-trigger/main.go
+++ b/pkg/cmd/teamcity-trigger/main.go
@@ -32,18 +32,18 @@ import (
 )
 
 func main() {
-	if len(os.Args) < 2 {
-		fmt.Fprintf(os.Stderr, "usage: %s BRANCH...\n", os.Args[0])
+	if len(os.Args) != 1 {
+		fmt.Fprintf(os.Stderr, "usage: %s\n", os.Args[0])
 		os.Exit(1)
 	}
-	branches := os.Args[1:]
 
+	branch := cmdutil.RequireEnv("TC_BUILD_BRANCH")
 	serverURL := cmdutil.RequireEnv("TC_SERVER_URL")
 	username := cmdutil.RequireEnv("TC_API_USER")
 	password := cmdutil.RequireEnv("TC_API_PASSWORD")
 
 	tcClient := teamcity.New(serverURL, username, password)
-	runTC(branches, func(buildID, branch string, opts map[string]string) {
+	runTC(func(buildID string, opts map[string]string) {
 		build, err := tcClient.QueueBuild(buildID, branch, opts)
 		if err != nil {
 			log.Fatalf("failed to create teamcity build (buildID=%s, branch=%s, opts=%+v): %s",
@@ -54,23 +54,21 @@ func main() {
 	})
 }
 
-func runTC(branches []string, queueBuild func(string, string, map[string]string)) {
+func runTC(queueBuild func(string, map[string]string)) {
 	importPaths := gotool.ImportPaths([]string{"github.com/cockroachdb/cockroach/pkg/..."})
 
-	for _, branch := range branches {
-		// Queue stress builds. One per configuration per package.
-		for _, opts := range []map[string]string{
-			{}, // uninstrumented
-			// The race detector is CPU intensive, so we want to run less processes in
-			// parallel. (Stress, by default, will run one process per CPU.)
-			//
-			// TODO(benesch): avoid assuming that TeamCity agents have eight CPUs.
-			{"env.GOFLAGS": "-race", "env.STRESSFLAGS": "-p 4"},
-		} {
-			for _, importPath := range importPaths {
-				opts["env.PKG"] = importPath
-				queueBuild("Cockroach_Nightlies_Stress", branch, opts)
-			}
+	// Queue stress builds. One per configuration per package.
+	for _, opts := range []map[string]string{
+		{}, // uninstrumented
+		// The race detector is CPU intensive, so we want to run less processes in
+		// parallel. (Stress, by default, will run one process per CPU.)
+		//
+		// TODO(benesch): avoid assuming that TeamCity agents have eight CPUs.
+		{"env.GOFLAGS": "-race", "env.STRESSFLAGS": "-p 4"},
+	} {
+		for _, importPath := range importPaths {
+			opts["env.PKG"] = importPath
+			queueBuild("Cockroach_Nightlies_Stress", opts)
 		}
 	}
 }

--- a/pkg/cmd/teamcity-trigger/main_test.go
+++ b/pkg/cmd/teamcity-trigger/main_test.go
@@ -21,11 +21,8 @@ import (
 
 func TestRunTC(t *testing.T) {
 	count := 0
-	runTC([]string{"master"}, func(buildID, branch string, opts map[string]string) {
+	runTC(func(buildID string, opts map[string]string) {
 		count++
-		if branch != "master" {
-			t.Errorf("unexpected branch %q", branch)
-		}
 		if pkg, ok := opts["env.PKG"]; ok {
 			if strings.Contains(pkg, "/vendor/") {
 				t.Errorf("unexpected package %s", pkg)

